### DISCUSSION TITLE
How to add another monitored interface?

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#2450] [Unsupported Versions] [answered]
In my VM I can easily add additional Nics.  When I add another Nic to the VM, how do I tell SO to add it as a monitored interface?  I looked through global.sls but didn't see anything about interfaces.  Do I have to run through the setup again?

**Comment by TOoSmOotH** [comment, marked as answer]
Just add the new interface to the bond

**Reply by innovate-support** [reply]
Can you be more specific on how to accomplish that? I don't know what you're referring to as 'the bond'.

**Reply by TOoSmOotH** [reply]
nmcli con add type ethernet ifname SOMEINTERFACE master bond0

**Reply by innovate-support** [reply]
Thanks so much.  That worked.  For those who use these discussion boards to get detailed answers, here's the steps I took to accomplish this.

1. Possibly snapshot your VM just in case you screw something up.
2. Add additional interface to VM and reboot just to make sure OS picks it up and applies drivers
3. Log into OS and run **`ifconfig`** command to see what name was applied to it.  My particular one got named **ens160**. Yours will probably be different.
4. Run **`nmcli con show`** command to view current interface bond-slaves
5. Run the following command to bond to the new interface, making sure your interface name is correct.  Mine was ens160
**`sudo nmcli con add type ethernet ifname ens160 master bond0`**
6. Run **`nmcli con show`** command again to confirm it was added as a bond-slave
7. You should now be able to see data being collected on the newly monitored interface
